README.rst

```diff
@@ -447,6 +447,62 @@ of the url, this will enable you to point all the API urls to
 your openwisp-network-topology API server's domain,
 example value: ``https://mytopology.myapp.com``.
 
+``Rest API``
```

Comment:
Just `Rest API` without the double quotes, which is only used for highlighting code, do not use if for non-code text please. Applies to all other headings.